harness/determined/_data_layer/_context.py

```diff
@@ -1,22 +1,25 @@
 from typing import Callable
 
 import determined as det
-from determined import _data_layer, horovod
+import determined._generic
```

Comment:
from determined import _generic

Prefer to import the final module as a single name.